nest/src/auth/auth.module.ts

```diff
@@ -5,16 +5,18 @@ import { JwtModule } from '@nestjs/jwt';
 import { UsersModule } from '../users/users.module';
 import { JwtStrategy } from './strategies/jwt.strategy';
 import { Intra42Strategy } from './strategies/intra42.strategy';
+import * as process from 'process';
+import { JwtRefreshStrategy } from './strategies/jwt-refresh.strategy';
 
 @Module({
   imports: [
     JwtModule.register({
-      secret: 'blablabla',
+      secret: process.env.JWT_SECRET,
```

Comment:
you can update the template of the .env file on discord, so that it's clear for everyone